sklearn/cluster/_k_means.py

```diff
@@ -648,7 +647,7 @@ def _init_centroids(X, k, init, random_state=None, x_squared_norms=None,
 
 
 class KMeans(TransformerMixin, ClusterMixin, BaseEstimator):
-    """K-Means clustering
+    """K-Means clustering.
```

Comment:
Other PRs are starting to put a newline after the triple quote right before the beginning of the docstring.

I prefer the convention here, but I am not sure what has been agreed upon.

Reply:
That was an issue in the validation script https://github.com/numpy/numpydoc/issues/241 the current version without the newline is the correct style.